test/extension/handlers/topologymutation/handler.go

```diff
@@ -220,14 +238,6 @@ func patchKubeadmConfigTemplate(ctx context.Context, k *bootstrapv1.KubeadmConfi
 		}
 		if version.Compare(machineDeploymentVersion, cgroupDriverPatchVersionCeiling) == -1 {
 			log.Info(fmt.Sprintf("Setting KubeadmConfigTemplate cgroup-driver to %q", cgroupDriverCgroupfs))
-			// Set the cgroupDriver in the InitConfiguration.
```

Comment:
Was this intentionally dropped?

Reply:
yes, InitConfiguration does not apply to worker machines

Reply:
Ahh, good catch!